docs/script.js

```diff
@@ -37,6 +38,85 @@ tabButtons.forEach((tabButton) => {
   });
 });
 
+// Tabs > Tabs Template
+const tabList_TabsTemp = document.querySelector("[aria-label='Tabs Template'");
```

Comment:
Some suggestions here.

- Since we're using `camelCase` for naming, let's follow this for consistency.
`tabList_TabsTemp` may change to `templateTabList`, `tabList_WinWTabs` may change to `windowTabList` etc. Up to you for the decision.
- (optional) The entire tab handling code block appears to be repetitive, we could rewrite it into 1 function and pass in the respective selectors, this way that function can be reused for handling several tabs without repeating and having a hard time naming the variables.